ndex2/client.py

```diff
@@ -1021,8 +1461,39 @@ def get_user_network_summaries(self, username, offset=0, limit=1000):
 
     def get_network_ids_for_user(self, username):
         """
+
         Get the network uuids owned by the user
 
+        .. code-block:: python
+
+		import ndex2.client as nc
+		import ndex2
+		import networkx as nx
+		import pandas as pd
+		import os
+
+		#For all networks in the current account, display a list of newtwork summaries 
+		#network_list = my_ndex.get_network_summaries_for_user("ENTER YOUR USERNAME HERE")
+		network_list = my_ndex.get_network_summaries_for_user("ENTER YOUR USERNAME HERE")
```

Comment:
Please add line showing how `my_ndex` is created. The only import needed is the one to import the Ndex2 client